backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -619,6 +619,17 @@
         </createTable>
     </changeSet>
 
+    <changeSet id="add-epoch_number" author="raviS">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="observation" columnName="epoch_number"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="observation">
+            <column name="epoch_number" type="int8" value="0"/>
+        </addColumn>
```

Comment:
We'll have to run a migration before shipping to prod that the existing values are not zero, but incremental.

Reply:
https://vertaai.atlassian.net/browse/VR-4912 indicates that epoch is optional